.env.sample

```
@@ -59,8 +62,10 @@ VITE_ANKR_API_KEY=XXX
 
 #ASG
 VITE_ASGARDEX_THORNAME=dx
+VITE_ASGARDEX_BROKER_URL=XXX
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_ASGARDEX_BROKER_URL key should go before the VITE_ASGARDEX_THORNAME key

(UnorderedKey)